src/pages/markets/NewMarket.tsx

```diff
@@ -0,0 +1,249 @@
+import { useMemo, useState } from 'react';
+import styled, { AnyStyledComponent } from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+
+import { STRING_KEYS } from '@/constants/localization';
+import { isMainnet } from '@/constants/networks';
+import { AppRoute } from '@/constants/routes';
+
+import {
+  useBreakpoints,
+  useDocumentTitle,
+  useGovernanceVariables,
+  useStringGetter,
+  useTokenConfigs,
+} from '@/hooks';
+
+import { breakpoints } from '@/styles';
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Button } from '@/components/Button';
+import { ContentSectionHeader } from '@/components/ContentSectionHeader';
+import { IconButton } from '@/components/IconButton';
+import { Icon, IconName } from '@/components/Icon';
+import { Link } from '@/components/Link';
+import { NewMarketForm } from '@/views/forms/NewMarketForm';
+
+import { MustBigNumber } from '@/lib/numbers';
+
+const StepItem = ({ step, subtitle, title }: { step: number; subtitle: string; title: string }) => (
+  <Styled.StepItem>
+    <Styled.StepNumber>{step}</Styled.StepNumber>
+    <Styled.Column>
+      <Styled.Title>{title}</Styled.Title>
+      <Styled.Subtitle>{subtitle}</Styled.Subtitle>
+    </Styled.Column>
+  </Styled.StepItem>
+);
+
+const NewMarket = () => {
+  const { isNotTablet } = useBreakpoints();
+  const { newMarketProposal } = useGovernanceVariables();
+  const navigate = useNavigate();
+  const [displaySteps, setDisplaySteps] = useState(true);
+  const stringGetter = useStringGetter();
+  const { chainTokenLabel } = useTokenConfigs();
+
+  useDocumentTitle('New Market');
+
+  const steps = useMemo(() => {
+    return [
+      {
+        step: 1,
+        title: stringGetter({ key: STRING_KEYS.ADD_MARKET_STEP_1_TITLE }),
+        subtitle: stringGetter({
+          key: STRING_KEYS.ADD_MARKET_STEP_1_DESCRIPTION,
+          params: {
+            HERE: (
+              <Styled.Link href={newMarketProposal.newMarketsMethodology}>
+                {stringGetter({ key: STRING_KEYS.HERE })}
+              </Styled.Link>
+            ),
+          },
+        }),
+      },
+      {
+        step: 2,
+        title: stringGetter({ key: STRING_KEYS.ADD_MARKET_STEP_2_TITLE }),
+        subtitle: stringGetter({ key: STRING_KEYS.ADD_MARKET_STEP_2_DESCRIPTION }),
+      },
+      {
+        step: 3,
+        title: stringGetter({ key: STRING_KEYS.ADD_MARKET_STEP_3_TITLE }),
+        subtitle: stringGetter({
+          key: STRING_KEYS.ADD_MARKET_STEP_3_DESCRIPTION,
+          params: {
+            REQUIRED_NUM_TOKENS: MustBigNumber(newMarketProposal?.initialDepositAmount)
+              .div(1e18)
+              .toFixed(isMainnet ? 0 : 18),
+            NATIVE_TOKEN_DENOM: chainTokenLabel,
+          },
+        }),
+      },
+    ];
+  }, [stringGetter, newMarketProposal, chainTokenLabel]);
+
+  return (
+    <Styled.Page>
```

Comment:
ummmm i don't love this padding difference between header section and content, do we need to keep it as a ContentSectionHeader or can we just make it part of the content so it's easier to match margin? (can be a todo)

<img width="1374" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/ae7de532-2c31-4a3e-acb7-45eb0d307daa">
<img width="664" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/e3f8c11b-3863-49a5-9a71-aaa48601db1e">

Reply:
Removed the horizontal margin on the Header section completely